### PR TITLE
- Added scene transition support

### PR DESCRIPTION
**Pull request by Markvanderuit**
- Added new functions to easily transition to other scenes
- Fixed some tests
- Checkstyle
- Tag?
  >Feature

### CODE REVIEW

File: resources/highscores.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+14774,Die gozer die beter dan jij was. Bart.
 11745,Mark
+9313,dusss
 7702,Menno
+5705,lage score lol
 2000,Mark die won
 1129,Menno
-1050,Mark die verloor
-100,test
-10,menno
```
Comment (mennoo1996) [inline resources/highscores.txt:10]: beetje mijn highscore verwijderen?

**Comment by mennoo1996** [comment]
Well done!

File: resources/highscores.txt (hunk 1 above)
Comment (Markvanderuit) [inline resources/highscores.txt:10]: Mijn score was zo goed, de parser moest m wel overwriten.

**Comment by geenen124** [comment]
Looks good!

**Pull request merged**